src/Codeception/Util/ActionSequence.php

```diff
@@ -54,7 +56,9 @@ public static function build()
 
     public function __call($action, $arguments)
     {
-        $this->actions[$action] = $arguments;
+        $key = "{$this->counter}.$action";
```

Comment:
Storing multiple pieces of information in a string is ugly, I think that it would be much more elegant to create `Action` object here.